### PR TITLE
Add 1.15 announcement and updates

### PR DESCRIPTION
**Pull request by morphis**

### CODE REVIEW

File: announcements/1.15.0.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+## New features & improvements
+
+### Common
+
+* All snaps are now based on the [`core20`](https://snapcraft.io/core20) snap, which removes the need to have older base snaps (for example, `core18`) installed on newer Ubuntu versions.
+
+### AMS
+
+* AMS now provides support for a [development mode](https://anbox-cloud.io/docs/exp/containers#dev-mode) for containers. This development mode mainly turns off status updates for the Anbox runtime inside a container, which usually cause AMS to stop the container.
```
[the remark this answers is not in the document]
Comment (morphis) [reply announcements/1.15.0.md:15]: Not only when an error occurs, also when the `anbox.service` systemd unit is stopped we send a signal which will turn the container down.

File: announcements/1.15.0.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+* With the new `cpu.limit_mode` configuration, the default strategy that AMS uses to assign CPU time to a container via the Linux kernel scheduler can be changed to use CPU core pinning instead. See [the documentation](https://anbox-cloud.io/docs/exp/performance#container-cpu-access) for more details and the caveats that each mode has.
+* If AMS is behind a load balancer, it can now be configured with the address of the load balancer via the `load_balancer.url` configuration item. This configuration will instruct containers to go through the load balancer instead of reaching out to the particular AMS instance that launched them.
+
+### Anbox
+
+* The [WebRTC streamer configuration](https://anbox-cloud.io/docs/ref/webrtc-streamer) can now be customised. Currently, only the mininum and the maximum bitrate can be configured, in addition to some options specific to [NVIDIA NvEnc](https://developer.nvidia.com/nvidia-video-codec-sdk).
```
Comment (ru-fu) [inline announcements/1.15.0.md:22]:
```suggestion
* The [WebRTC streamer configuration](https://anbox-cloud.io/docs/ref/webrtc-streamer) can now be customised. Currently, only the minimum and the maximum bit rate can be configured, in addition to some options specific to [NVIDIA NvEnc](https://developer.nvidia.com/nvidia-video-codec-sdk).
```

File: announcements/1.15.0.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+### Android
+
+* Added Android security updates for August 2022 (see [Android Security Bulletin - August 2022](https://source.android.com/docs/security/bulletin/2022-08-01) for more information).
+* Updated the Android WebView to [103.0.5060.129](https://chromereleases.googleblog.com/2022/07/chrome-for-android-update_01510389319.html).
+* Updated the Mesa graphics driver stack to [22.0.5](https://docs.mesa3d.org/relnotes/22.0.5.html).
+* The `ashmem` kernel module is now optional for all supported Android versions. Anbox Cloud will drop support for `ashmem` with the 1.18 release.
+
+### Anbox Stream Gateway / Agent
+
+* Updated dqlite to its latest [1.11.1 release](https://github.com/canonical/dqlite/releases/tag/v1.11.1).
```
Comment (ru-fu) [inline announcements/1.15.0.md:39]:
```suggestion
* Updated Dqlite to its latest [1.11.1 release](https://github.com/canonical/dqlite/releases/tag/v1.11.1).
```

File: announcements/1.15.0.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+  * Set the WebRTC log level to gain low-level insight (`--log-level=<level>`).
+
+## Known issues
+
+n/a
+
+## Bug fixes
+
+* AC-1036 A native crash happens when streaming from the camera on Safari (iOS device)
+* AC-1034 Fix race condition between `init.ranchu.rc` and SurfaceFlinger causing a crash
```
Comment (ru-fu) [inline announcements/1.15.0.md:64]: Can you update the capitalisation of SurfaceFlinger in the wordlist?
This capitalisation here is correct. That might cause other failures, but I can fix them on the 1.15 branch.

File: howto/update/upgrade-anbox.md [hunk 5]
```diff
@@ -29,6 +29,8 @@ The deployed Juju charms need to be upgraded next.
 
 - Starting with the 1.14 release, all charms come from [Charmhub](https://charmhub.io) and use the concept of [channels](https://snapcraft.io/docs/channels) to track particular versions. The instructions below address how to upgrade from a 1.13.x release, where charms were still from the old Juju charm store. The `--switch --channel=1.14/stable` arguments instruct Juju to switch to the [Charmhub](https://charmhub.io) version of the charm and track the right channel.
 
+- Starting with the 1.15 release AMS will enforce TLS 1.3 on its HTTPS endpoint. Images older than 1.15.0 will fail to reach AMS in this case. To still allow older images to work with the newer AMS you can temporarily enable TLS 1.2 support again in AMS by setting the `force_tls12` [configuration option of the AMS charm](https://charmhub.io/ams/configure?channel=1.15/stable#force_tls12).
```
Comment (ru-fu) [inline howto/update/upgrade-anbox.md:32]:
```suggestion
- Starting with the 1.15 release, AMS enforces TLS 1.3 on its HTTPS endpoint. Images older than 1.15.0 will fail to reach AMS in this case. To still allow older images to work with AMS, you can temporarily enable TLS 1.2 support again in AMS by setting the `force_tls12` [configuration option of the AMS charm](https://charmhub.io/ams/configure?channel=1.15/stable#force_tls12).
```

File: index.md [hunk 6]
```diff
@@ -13,13 +13,12 @@ You should consider using Anbox Cloud if you want to deliver mobile applications
...
 
-Along with bug fixes and general improvements, Anbox Cloud 1.14 includes the following changes:
+Along with bug fixes and general improvements, Anbox Cloud 1.15 includes the following changes:
 
-* All charms have been migrated to [Charmhub](https://charmhub.io).
-* Anbox Cloud now includes Anbox LXD images based on Ubuntu 22.04.
-* Hook support for AMS applications.
-* Vulkan support for AMD and Intel GPUs.
-* Multi-node support for the Anbox Cloud Appliance (experimental).
+* The Anbox Cloud Appliance now supports Ubuntu 22.04
```
Comment (ru-fu) [inline index.md:18]:
```suggestion
* Support for Ubuntu 22.04 in the Anbox Cloud Appliance
```

File: index.md [hunk 7]
```diff
@@ -13,13 +13,12 @@ You should consider using Anbox Cloud if you want to deliver mobile applications
...
+* [Out-of-band data version 2](https://discourse.ubuntu.com/t/exchange-out-of-band-data/21834)
+* Audio-only streaming support
+* Development mode support in AMS for containers
```
Comment (ru-fu) [inline index.md:21]:
```suggestion
* Support for a development mode for containers in AMS
```

File: release-notes.md [hunk 8]
```diff
@@ -3,6 +3,51 @@
...
+* All snaps are now based on the [`core20`](https://snapcraft.io/core20) snap.
+* AMS now provides support for a [development mode](https://anbox-cloud.io/docs/exp/containers#dev-mode).
+* TLS 1.3 is now enforced by default by AMS.
+* New `cpu.limit_mode` AMS configuration option allows to enable CPU core pinning.
+* Configuration of an external load balancer for containers spawned by AMS.
+* The [WebRTC streamer configuration](https://anbox-cloud.io/docs/ref/webrtc-streamer) can now be customised.
+* A new HTTP `/1.0/platform` API allows runtime configuration of the currently loaded platform.
+* Anbox now uses the recently released [LXC 5.0](https://discuss.linuxcontainers.org/t/lxc-5-0-lts-has-been-released/14381).
+* The WebRTC streamer now allows streaming only audio, only video or neither of them.
+* Anbox now uses [`jemalloc`](https://github.com/jemalloc/jemalloc) as memory allocator
```
Comment (ru-fu) [inline release-notes.md:21]:
```suggestion
* Anbox now uses [`jemalloc`](https://github.com/jemalloc/jemalloc) as memory allocator.
```

File: release-notes.md [hunk 9]
```diff
@@ -3,6 +3,51 @@
...
+* A new OOB version 2 implementation allows bi-directional communication with the Anbox container through the use of data channels
```
Comment (ru-fu) [inline release-notes.md:22]:
```suggestion
* A new OOB version 2 implementation allows bi-directional communication with the Anbox container through the use of data channels.
```

File: release-notes.md [hunk 10]
```diff
@@ -3,6 +3,51 @@
...
+* Added Android security updates for August 2022 (see [Android Security Bulletin - August 2022](https://source.android.com/docs/security/bulletin/2022-08-01) for more information).
+* Updated the Android WebView to [103.0.5060.129](https://chromereleases.googleblog.com/2022/07/chrome-for-android-update_01510389319.html).
+* Updated the Mesa graphics driver stack to [22.0.5](https://docs.mesa3d.org/relnotes/22.0.5.html).
+* The `ashmem` kernel module is now optional for all supported Android versions.
+* Updated dqlite to its latest [1.11.1 release](https://github.com/canonical/dqlite/releases/tag/v1.11.1).
```
Comment (ru-fu) [inline release-notes.md:27]:
```suggestion
* Updated Dqlite to its latest [1.11.1 release](https://github.com/canonical/dqlite/releases/tag/v1.11.1).
```

**Comment by ru-fu** [comment]
No need to add OOB to the wordlist, that will already be added through https://github.com/anbox-cloud/discourse-docs/pull/153

**Pull request merged**